beacon-chain/rpc/prysm/v1alpha1/validator/status_test.go

```diff
@@ -603,15 +603,15 @@ func TestActivationStatus_OK(t *testing.T) {
 }
 
 func TestOptimisticStatus(t *testing.T) {
-	server := &Server{HeadFetcher: &mockChain.ChainService{}, TimeFetcher: &mockChain.ChainService{}}
-	err := server.optimisticStatus(context.Background())
-	require.NoError(t, err)
-
 	params.SetupTestConfigCleanup(t)
```

Comment:
this looks like a bigger diff than it is, just moved `SetupTestConfigCleanup` to the top of the function.

Reply:
this was the merge conflict @terencechain, diff looks good now